test/ng/compileSpec.js

```diff
@@ -231,11 +231,26 @@ describe('$compile', function() {
 
   describe('svg namespace transcludes', function() {
     // this method assumes some sort of sized SVG element is being inspected.
-    function assertIsValidSvgCircle(elem) {
+    function assertIsValidSvgCircle(elem, possiblyUnsizedParent) {
       expect(isUnknownElement(elem)).toBe(false);
       expect(isSVGElement(elem)).toBe(true);
+
       var box = elem.getBoundingClientRect();
-      expect(box.width === 0 && box.height === 0).toBe(false);
+      var hasWidthOrHeight = (box.width !== 0) || (box.height !== 0);
+
+      if (possiblyUnsizedParent && !hasWidthOrHeight) {
```

Comment:
I was not sure what the purpose of the test was (not to mention it has a comment that says it might be redundant), so I am open to alternative suggestions.

TBH, I am not too familiar with SVG, so I don't really know why MS Edge behaves differently than other browsers (although it theoretically supports `foreignObject`). It isn't Angular-related, btw, because the same behavior can be reproduced with "raw" HTML.